embeddings/app/clipL14/loadModel.py

```diff
@@ -0,0 +1,12 @@
+## This script downloads the clip model for embeddings service.
+
+import torch
+import torchvision.transforms as transforms
+import clip
+
+device = "cuda" if torch.cuda.is_available() else "cpu"
+model, preprocess = clip.load('ViT-L/14', device)
+
+#model.save('/code/app/clip-ViT-L-14.model')
+# Running into AttributeError: 'CLIP' object has no attribute 'save'
+# No save method at https://github.com/openai/CLIP
```

Comment:
Is this file being called? - Can't see it in the Dockerfile

Reply:
No !

Reply:
Could you please remove the file then?

Reply:
The answer added below was actually intended for this comment : 
"I kept the file with the reason why this wasn't possible and to maintain the existing structure."

Reply:
done